lib/DDG/Spice/TwitchStreams.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::TwitchStreams;
+use strict;
+use DDG::Spice;
+
+name "TwitchStreams";
+source "http://www.twitch.tv";
+description "Get Twitch Stream info";
+primary_example_queries "twitch streams dota 2";
+category "special";
+topics "special_interest";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/TwitchStreams/TwitchStreams.pm";
+attribution github => ["https://github.com/dwaligon", "Jason O'Donnell"];
+
+triggers startend => ('twitch streams',
+		              'streams twitch',
+		              'twitchtv streams');
```

Comment:
@Dwaligon yup that's perfectly fine. The red alert is just to let you know as a developer that some IA ran into a problem (we should find a way to make it more clear which IA it was) -- that won't appear in production though.

Separately we should updated the TVMaze code not to try and return a result for that query.